examples/usecases/accelerate-training-of-large-embedding-tables-by-LazyAdam.ipynb

```diff
@@ -0,0 +1,695 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.                             item_tower=ml.InputBlockV2(schema.select_by_tag(Tags.ITEM), embeddings=item_embeddings).connect(ml.MLPBlock([64])), 
</pre><p>The embeddings need to be passed in as the keyword argument <code>categorical</code> since the changes made in https://github.com/NVIDIA-Merlin/models/pull/754</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>fixed. thanks!</p>